README.md

```diff
@@ -1 +1,5 @@
-# Review-one-of-your-past-projects-Microverse
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Your README.md file is incomplete. This is something to point out when giving a code review. Here, you can see that there is no _Built with_ and _Getting started_ sections. Also, the social handle are not there as well. Kindly point these out when giving a code review. 😄